documentation/API-design-guidelines.md

```diff
@@ -1183,16 +1198,16 @@ In order to ease developer adoption, the pattern for Resource-based event subscr
 
 | operation | path | description |
 | ----- |	-----  |	 -----  | 
-| POST | `/event-subscriptions` |  Operation to request a event subscription.     |
-| GET | `/event-subscriptions` |  Operation to retrieve a list of event subscriptions - could be an empty list.  eg. `GET /event-subscriptions?type=ROAMING_STATUS&ExpireTime.lt=2023-03-17` |
-| GET | `/event-subscriptions/{eventSubscriptionsId}` | Operation to retrieve a event subscription |
-| DELETE | `/event-subscriptions/{eventSubscriptionsId}` | Operation to delete a event subscription |
+| POST | `/subscriptions` |  Operation to request a event subscription.     |
+| GET | `/subscriptions` |  Operation to retrieve a list of event subscriptions - could be an empty list.  eg. `GET /subscriptions?type=ROAMING_STATUS&ExpireTime.lt=2023-03-17` |
+| GET | `/subscriptions/{subscriptionsId}` | Operation to retrieve a event subscription |
+| DELETE | `/subscriptions/{subscriptionsId}` | Operation to delete a event subscription |
 
 
 Note on the operation path:
-The recommended pattern is to use `/event-subscriptions` path for the subscription operation. But API design team, for specific case, has the option to append `/event-subscriptions` path with a prefix (e.g. `/roaming/event-subscriptions` and `/connectivity/event-subscriptions`). The rationale for using this alternate pattern should be explicitly provided (e.g. the notification source for each of the supported events may be completely different, in which case separating the implementations is beneficial). 
+The recommended pattern is to use `/subscriptions` path for the subscription operation. But API design team, for specific case, has the option to append `/subscriptions` path with a prefix (e.g. `/roaming/subscriptions` and `/connectivity/subscriptions`). The rationale for using this alternate pattern should be explicitly provided (e.g. the notification source for each of the supported events may be completely different, in which case separating the implementations is beneficial). 
```

Comment:
Wouldn't this kind of an (optional) categorization or extended information be more better in the type `org.camaraproject.<optional-category or extension><api-name>.<api-version>.<event-name>`rather than in the path? WDYT?

Other than that I am almost ready to approve this review :)

Reply:
@shilpa-padgaonkar Not sure to follow you. You propose to update the `type` attribute with an additional `<optional-category or extension>` but this point in the guideline is not about the event type but the `/subscriptions` endpoint. 

What we say here is for explicit subscription pattern we recommend a generic `/subscriptions` endpoint but for some API with several capabilities we allow to 'specialize' this subscription endpoint (and provide several) to make it very explicit for the developer.

Reply:
@bigludo7 Then may be the examples were misleading to me :)  Because the examples shown here at least to me seem as some sort of additional info /categorization about the event. If this is not the case, you can ignore my comment :)

Reply:
Thanks @shilpa-padgaonkar  - indeed this is not related to the event directly but to the subscription.